.github/actions/check_docstrings/action.yml

```diff
@@ -0,0 +1,56 @@
+name: Check Docstrings
+inputs:
+    python-version:
+        description: 'The version of Python to use for the workflow.'
+        default: '3.10'
```

Comment:
I would suggest using 3.11 already has it already entered "security only changes" phase:
https://devguide.python.org/versions/